src/clr-angular/forms/tests/container.spec.ts

```diff
@@ -127,25 +127,25 @@ function fullSpec(description, testContainer, directives: any | any[], testCompo
     });
 
     it('adds the .clr-row class to the host on non-vertical layouts', () => {
-      expect(containerEl.classList).not.toContain('clr-row');
-      layoutService.layout = Layouts.HORIZONTAL;
-      fixture.detectChanges();
       expect(containerEl.classList).toContain('clr-row');
+      layoutService.layout = Layouts.VERTICAL;
+      fixture.detectChanges();
+      expect(containerEl.classList).not.toContain('clr-row');
       layoutService.layout = Layouts.COMPACT;
       fixture.detectChanges();
       expect(containerEl.classList).toContain('clr-row');
     });
 
     it('computes the error class for the control container', () => {
```

Comment:
Personal nit, not a blocker.
Instead of `computes` could this be changed to `adds`? Reading that makes more sense to me for what is being tested.

Reply:
Sure